src/chains.rs

```diff
@@ -267,8 +267,15 @@ impl ChainItemKind {
 }
 
 impl Rewrite for ChainItem {
+    // TODO impl rewrite_result after rebase
```

Comment:
Just wondering what this comment is referencing. Can this be removed?

Reply:
Sure, since this pr also implements rewrite_result for ChainItem, I removed the comment.